tests/Doctrine/Tests/DBAL/Functional/ExceptionTest.php

```diff
@@ -274,7 +275,8 @@ public function testConnectionExceptionSqLite($mode, $exceptionClass)
     public function getSqLiteOpenConnection()
     {
         return array(
-            array(0000, '\Doctrine\DBAL\Exception\ConnectionException'),
+            // mode 0 is considered read-only on Windows
+            array(0000, '\\' !== DIRECTORY_SEPARATOR ? '\Doctrine\DBAL\Exception\ConnectionException' : '\Doctrine\DBAL\Exception\ReadOnlyException'),
```

Comment:
This looks like we have an inconsistency here between different OS. IMO we should rather try to fix that to throw the same exception on all OS, otherwise it is not abstracted very well.